dbms/src/Flash/Pipeline/Schedule/tests/gtest_task_scheduler.cpp

```diff
@@ -295,12 +307,14 @@ try
             tasks.push_back(std::make_unique<SimpleWaitingTask>(waiter));
         submitAndWait(tasks, waiter);
     }
+    FAILPOINT_TEST_END
 }
 CATCH
 
-TEST_F(TaskSchedulerTestRunner, test_memory_trace)
+TEST_F(TaskSchedulerTestRunner, testMemoryTrace)
 try
 {
+    FAILPOINT_TEST_BEGIN
```

Comment:
I suggest not modifying the existing unit tests. Each unit test has its own testing purpose. For example,  `testMemoryTrace` is used to test whether the memory tracker is correctly set up. The `Failpoint` may skip the detection of the memory tracker.